mirage/views/search.ts

```diff
@@ -66,6 +100,21 @@ const resourceMetadataByType: Partial<Record<OsfmapResourceTypes, any>> = {
             '@value': faker.lorem.sentence(),
             '@type': rdfString,
         }],
+        funder: [_shareOrganizationField()],
+        hasOsfAddon: [
+            {
+                prefLabel: [{
+                    '@value': 'OSF Storage', // CR question: do we need to include osf storage as addons?
```

Comment:
As of now probably not, since it's not an optional thing right now. If we ever refactored it to be an addon, then maybe.